### DISCUSSION TITLE
Is Xamarin Android expected to remain compatible with the latest Android versions through 2025?

### DISCUSSION THREAD

**Discussion by IniyansHub** [CommunityToolkit/Maui#1855] [General]
Should I prioritize the migration of my Xamarin app's Android part to MAUI, considering I have already completed the migration of the iOS part, or can I delay the Android migration until 2025?

**Comment by CliffAgius** [comment]
Xamarin is officially not supported as of May 2024 you can [Read about it here](https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin)
